src/hash/rpo/mod.rs

```diff
@@ -294,6 +294,28 @@ impl Rpo256 {
         <Self as ElementHasher>::hash_elements(elements)
     }
 
+    // MERGE WITH DOMAIN
+    // --------------------------------------------------------------------------------------------
+
+    /// Returns a hash of two digests and a domain separator.
```

Comment:
nit: It might be clearer to say "domain identifier". @Al-Kindi-0 what do you think?

Reply:
Sounds better to me also.

Reply:
done

Reply:
I'm still seeing it as "domain separator" in the doc comment